.github/workflows/update_python_dependencies.yml

```
@@ -0,0 +1,38 @@
+name: Update Python Dependencies
+
+on:
+    workflow_dispatch:
+
+jobs:
+    update_dependencies:
+        runs-on: ubuntu-latest
+        steps:
+            - name: "Checkout"
+              uses: actions/checkout@v3
+              with:
+                  ref: main
+                  fetch-depth: 0
+            - name: Set up Python 3.8
+              id: py
+              uses: actions/setup-python@v2
+              with:
+                  python-version: 3.8
```

±: Matching the above change:
```suggestion
            - name: Set up Python ${{ inputs.python-version }}
              id: py
              uses: actions/setup-python@v2
              with:
                  python-version: ${{ inputs.python-version }}
```